goal_frame: "/map"
odom_topic: "odom"
move_base_action: "move_base"
publish_frequency: 1.0
traveling_timeout: 240
nav_goal_idle_time: 5
nav_error_idle_time: 10
max_goal_misses_in_a_row: 5

docking_goals:
 - docking1: [0.8, -0.2, 3.14]
 - docking2: [12.87, 5.7, 0]
 - docking3: [-7.8, -7.1, 2.32]
 - docking4: [13.34, -1.11, -0.157]
# - docking1: [-11.25, -12.16, 1.57] #SIMULATION DOCKINGS
# - docking2: [20, -6, 1.57]         #SIMULATION DOCKINGS
# - docking3: [3, 3, 1.57]           #SIMULATION DOCKINGS
# - docking4: [-9.7, -19.36, -1.67]    #SIMULATION DOCKINGS

nav_goals:
 - door1: [9.0, -6.0, 1.57]
 - lab1_A: [7.1, 6.8, 1.97]
 - lab1_B: [10.7, 1.2, -0.18]
 - door2: [17.7, -5.7, 2.652]
 - lab2: [18.1, 0.37, -2.642]
 - lab_car: [-7, -5, -2.207]
 - lab3_A: [-12.25, -14.16, 1.57]
 - lab3_B: [-11.06, -18.6, -0.18]
 - lab3_C: [-15.317, -15.686, -2.207]
# - not_reachable: [9, -8.0, 0.0]

autodocking_action: "dock_drive_action"
kobuki_sensors_topic: "mobile_base/sensors/core"
autodocking_timeout: 100
max_autodocking_misses_in_a_row: 5
min_battery_value: 140 # [0.1V] calibrate the battery (minimum voltage level)
max_battery_value: 161 # [0.1V] calibrate the battery, check sensor status if battery is full
battery_low_percentage: 60
battery_full_percentage: 95
kobuki_buttons_topic: "mobile_base/events/button"

## Navigation mode select if there is a master who send goals or
## the robot should navigate random to the nav_goals
random_mode : false
remote_mode : false
voice_mode : true
simulation_mode : true
    
allow_driving_start_hour: 8
allow_driving_start_min: 30
allow_driving_end_hour: 17
allow_driving_end_min: 30
